R/source.R

```diff
@@ -36,7 +36,15 @@ source_file <- function(path, env = test_env(), chdir = TRUE,
   if (wrap) {
     invisible(test_code(NULL, exprs, env))
   } else {
-    invisible(eval(exprs, env))
+    withCallingHandlers(
+      invisible(eval(exprs, env)),
+      error = function(err) {
+        abort(
+          paste0("Failed to source ", encodeString(path, quote = '"')),
```

Comment:
```suggestion
          paste0("Can't source ", encodeString(path, quote = '"')),
```

Reply:
I wonder if I like "failed" when the problem is not directly related to the argument supplied.

But maybe this should actually just be `In: `

Reply:
oh yeah looks like a good candidate for an info "In" bullet